### PR TITLE
CIS compliance: mssql

### PR DESCRIPTION
**Pull request by t-young31**
- Adds a missed CIS task for the MSSQL server
- Updates the CIS doc accordingly, inc. a couple of TODO->Y that got missed

### CODE REVIEW

File: CIS-Adherence.md [hunk 1]
```diff
@@ -99,14 +99,14 @@ This layer deploys components required to ingest data, transform data, and save
 | Azure Resource | CIS Reference | Adherence | Notes |
 |--|--|--|--|
 | Azure SQL Server:<br/> `sql-server-features-suffix` | [feature-data-store.tf](./infrastructure/core/feature-data-store.tf) | |
-| | `CIS 4.1.1`: Ensure auditing is set to 'on' | TODO | https://github.com/UCLH-Foundry/FlowEHR/issues/172 |
+| | `CIS 4.1.1`: Ensure auditing is set to 'on' | Y | https://github.com/UCLH-Foundry/FlowEHR/issues/172 |
```
Comment (t-young31) [inline CIS-Adherence.md:102]: set in https://github.com/UCLH-Foundry/FlowEHR/pull/210

**Comment by t-young31** [comment]
:robot: pr-bot :robot:

:runner: Running tests: https://github.com/UCLH-Foundry/FlowEHR/actions/runs/4597557261 (with refid `9a0d8892`)

(in response to [this comment](https://github.com/UCLH-Foundry/FlowEHR/pull/231#issuecomment-1494371888) from @t-young31)

**Review by jjgriff93 — APPROVED** [review]

**Comment by t-young31** [comment]
:robot: pr-bot :robot:

:runner: Running tests: https://github.com/UCLH-Foundry/FlowEHR/actions/runs/4606084836 (with refid `9a0d8892`)

(in response to [this comment](https://github.com/UCLH-Foundry/FlowEHR/pull/231#issuecomment-1495629875) from @t-young31)

**Comment by t-young31** [comment]
:robot: pr-bot :robot:

:runner: Running tests: https://github.com/UCLH-Foundry/FlowEHR/actions/runs/4606807020 (with refid `9a0d8892`)

(in response to [this comment](https://github.com/UCLH-Foundry/FlowEHR/pull/231#issuecomment-1495740259) from @t-young31)

**Comment by t-young31** [comment]
:robot: pr-bot :robot:

Destroying environment (with refid 9a0d8892)

(in response to [this comment](https://github.com/UCLH-Foundry/FlowEHR/pull/231#issuecomment-1495747396) from @t-young31)

**Comment by t-young31** [comment]
:robot: pr-bot :robot:

:runner: Running tests: https://github.com/UCLH-Foundry/FlowEHR/actions/runs/4607023002 (with refid `9a0d8892`)

(in response to [this comment](https://github.com/UCLH-Foundry/FlowEHR/pull/231#issuecomment-1495776805) from @t-young31)

**Comment by t-young31** [comment]
:robot: pr-bot :robot:

:runner: Running tests: https://github.com/UCLH-Foundry/FlowEHR/actions/runs/4607427518 (with refid `9a0d8892`)

(in response to [this comment](https://github.com/UCLH-Foundry/FlowEHR/pull/231#issuecomment-1495847759) from @t-young31)

**Comment by t-young31** [comment]
:robot: pr-bot :robot:

Destroying environment (with refid 9a0d8892)

(in response to [this comment](https://github.com/UCLH-Foundry/FlowEHR/pull/231#issuecomment-1495926766) from @t-young31)

**Comment by t-young31** [comment]
/test-force-approve

Destroy failed with a transient issue: https://github.com/UCLH-Foundry/FlowEHR/actions/runs/4607427518/jobs/8142225543

**Comment by t-young31** [comment]
:robot: pr-bot :robot:

:white_check_mark: Marking tests as complete (for commit cda48789ac3f7259a900f5e3838882534cecf299)

(in response to [this comment](https://github.com/UCLH-Foundry/FlowEHR/pull/231#issuecomment-1497192174) from @t-young31)

**Pull request merged**